<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Sequence contamination filtering, Genome assembly, Sequence assembly validation, Sequencing quality control, Visualisation, Sequence assembly visualisation, Sequence composition calculation, Statistical calculation</td>
      <td>2021-03-24</td>
      <td>2024-09-18</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bandage_image, shovill, fastqc, bandage_info, upload1, fastp, quast</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>6791</td>
      <td>10915</td>
      <td>3.783333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial Resistance</td>
      <td>Genome assembly, Sequencing error detection, Sequence assembly visualisation, De-novo assembly, Filtering, Mapping assembly, Scatter plot plotting, Sequence assembly validation, Visualisation, Cross-assembly, Sequence alignment, Sequence composition calculation, Box-Whisker plot plotting, Sequencing quality control, Read mapping, Sequence contamination filtering, Statistical calculation</td>
      <td>2021-03-24</td>
      <td>2024-03-13</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bandage_image, polypolish, filtlong, fastqc, nanoplot, upload1, bwa_mem2, fastp, porechop, CONVERTER_bz2_to_uncompressed, quast, flye</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>7215</td>
      <td>11067</td>
      <td>3.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Variant calling, Genome assembly, Sequence assembly visualisation, Sequence assembly validation, Visualisation, Sequence trimming, Local alignment, Sequence file editing, Data handling, Sequence alignment analysis, Primer removal, Formatting, Sequence composition calculation, Sequencing quality control, Read mapping, Read pre-processing, Sequence contamination filtering, Statistical calculation</td>
      <td>2024-10-08</td>
      <td>2024-10-08</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bandage_image, seqtk_subseq, megahit_contig2fastg, cutadapt, random_lines1, tp_cat, megahit, fastqc, bg_uniq, collection_column_join, bandage_info, metaspades, coverm_contig, bowtie2, filter_tabular, quast, bamtools, ngsutils_bam_filter</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>6</td>
      <td>14</td>
      <td>7.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Genome assembly, Coding region prediction, Sequence composition calculation, Genome annotation, Sequence assembly validation, Validation, Sequencing quality control, Gene prediction, Visualisation, Aggregation, Statistical calculation</td>
      <td>2017-10-11</td>
      <td>2024-03-13</td>
      <td>23</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>unicycler, fastqc, multiqc, prokka, quast</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, HyPhy HIV NGS Tools, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2155238</td>
      <td>5948759</td>
      <td>4.583333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Read mapping, Sequence alignment, Sequence visualisation, Genome visualisation, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cat1, tp_sed_tool, lastz_wrapper_2, jbrowse, circos_interval_to_tiles, tp_grep_tool, circos, bedtools_complementbed, collapse_dataset, bedtools_intersectbed, upload1, datamash_ops, tp_replace_in_column, addValue, Filter1, tp_replace_in_line, random_lines1, tp_sort_header_tool, Grep1, fasta_compute_length, mergeCols1, Cut1, fasta_filter_by_length, tp_cat, join1, bedtools_sortbed, circos_aln_to_links</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6047</td>
      <td>12192</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Expression analysis, Cross-assembly, Statistical calculation, Taxonomic classification</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>kraken2, upload1, recentrifuge, est_abundance</td>
      <td>UseGalaxy.eu</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>267</td>
      <td>352</td>
      <td>3.366667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_rstudio, upload1, raxml</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1378</td>
      <td>2078</td>
      <td>2.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Variant calling, Antimicrobial resistance prediction, Phylogenetic tree visualisation, Phylogenetic tree generation</td>
      <td>2022-03-16</td>
      <td>2024-07-12</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>samtools_view, tp_replace_in_line, tp_sed_tool, bcftools_consensus, snippy, snp_sites, collapse_dataset, trimmomatic, tp_cat, tp_easyjoin_tool, tp_grep_tool, Grep1, tb_variant_filter, addName, tb_profiler_profile, upload1, __MERGE_COLLECTION__, snp_dists</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1610</td>
      <td>2504</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Coding region prediction, Gene prediction, Genome visualisation, Genome annotation</td>
      <td>2018-03-06</td>
      <td>2024-08-08</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>prokka, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2160036</td>
      <td>5956792</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome visualisation, Read mapping, Genome annotation</td>
      <td>2024-01-23</td>
      <td>2024-09-27</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, Grep1, tbl2gff3, bakta, upload1, bowtie2, staramr_search</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1181</td>
      <td>1633</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-08-08</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, create_or_update, create_account, iframe, list_organism</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1916586</td>
      <td>5151189</td>
      <td>5.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Sequence motif recognition, Genome assembly, Genome visualisation, Protein feature detection, Genome annotation, Multilocus sequence typing, Nucleic acid feature detection, Scaffolding, Structural variation detection</td>
      <td>2024-02-01</td>
      <td>2024-09-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>integron_finder, jbrowse, tp_tail_tool, bakta, tbl2gff3, upload1, tp_replace_in_column, Grouping1, isescan, plasmidfinder</td>
      <td>UseGalaxy.eu</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2301</td>
      <td>3039</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>Multiple sequence alignment, Sequence alignment analysis, Transcriptome assembly, Coding region prediction, Database search, De-novo assembly, Phylogenetic tree generation</td>
      <td>2022-09-08</td>
      <td>2024-03-18</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, cat1, bg_diamond_view, tp_split_on_column, collapse_dataset, rbc_mafft, tab2fasta, regexColumn1, gops_intersect_1, join1, rapidnj, bg_diamond, orfipy, Add_a_column1, bg_diamond_makedb, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1051</td>
      <td>1608</td>
      <td>3.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Read pre-processing, Sequence trimming, Primer removal, Transposon prediction</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, transit_gumbel, cutadapt, tp_easyjoin_tool, bg_find_subsequences, tp_sort_header_tool, deeptools_bam_coverage, bowtie_wrapper, __EXTRACT_DATASET__, Add_a_column1, gff_to_prot, Filter1</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2150876</td>
      <td>5941722</td>
      <td>4.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Coding region prediction, Gene prediction, Genome visualisation, Genome annotation</td>
      <td>2018-03-06</td>
      <td>2024-08-08</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>prokka, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2160036</td>
      <td>5956800</td>
      <td>4.316667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome visualisation, Read mapping, Genome annotation</td>
      <td>2024-01-23</td>
      <td>2024-09-27</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, Grep1, tbl2gff3, bakta, upload1, bowtie2, staramr_search</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1181</td>
      <td>1633</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-08-08</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, create_or_update, create_account, iframe, list_organism</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>1916586</td>
      <td>5151198</td>
      <td>5.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Sequence motif recognition, Genome assembly, Genome visualisation, Protein feature detection, Genome annotation, Multilocus sequence typing, Nucleic acid feature detection, Scaffolding, Structural variation detection</td>
      <td>2024-02-01</td>
      <td>2024-09-08</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>integron_finder, jbrowse, tp_tail_tool, bakta, tbl2gff3, upload1, tp_replace_in_column, Grouping1, isescan, plasmidfinder</td>
      <td>UseGalaxy.eu</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2301</td>
      <td>3039</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Variant calling, Phylogenetic tree visualisation, Genome visualisation, Phylogenetic tree generation</td>
      <td>2018-02-26</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, snippy</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>12499</td>
      <td>20722</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial Resistance</td>
      <td>Genome assembly, Sequencing error detection, Sequence assembly visualisation, De-novo assembly, Filtering, Mapping assembly, Scatter plot plotting, Sequence assembly validation, Visualisation, Cross-assembly, Sequence alignment, Sequence composition calculation, Box-Whisker plot plotting, Sequencing quality control, Read mapping, Sequence contamination filtering, Statistical calculation</td>
      <td>2021-03-24</td>
      <td>2024-03-13</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bandage_image, polypolish, filtlong, fastqc, nanoplot, upload1, bwa_mem2, fastp, porechop, CONVERTER_bz2_to_uncompressed, quast, flye</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>7215</td>
      <td>11067</td>
      <td>3.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Read mapping, Sequence alignment, Sequence visualisation, Genome visualisation, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cat1, tp_sed_tool, lastz_wrapper_2, jbrowse, circos_interval_to_tiles, tp_grep_tool, circos, bedtools_complementbed, collapse_dataset, bedtools_intersectbed, upload1, datamash_ops, tp_replace_in_column, addValue, Filter1, tp_replace_in_line, random_lines1, tp_sort_header_tool, Grep1, fasta_compute_length, mergeCols1, Cut1, fasta_filter_by_length, tp_cat, join1, bedtools_sortbed, circos_aln_to_links</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6047</td>
      <td>12192</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Sequence read processing, DNA barcoding, Taxonomic classification, Sequencing quality control, Visualisation, Phylogenetic analysis, Sequence clustering</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>krona-text, mothur_merge_files, mothur_count_seqs, mothur_screen_seqs, mothur_summary_seqs, mothur_classify_otu, mothur_unique_seqs, mothur_make_biom, mothur_classify_seqs, mothur_pre_cluster, metaphlan2, metaphlan2krona, humann2_renorm_table, mothur_make_group, humann2_regroup_table, mothur_filter_seqs, mothur_align_seqs, humann2, mothur_make_shared, mothur_cluster_split, taxonomy_krona_chart</td>
      <td>UseGalaxy.be</td>
      <td>UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3689</td>
      <td>4998</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Sequence composition calculation, Sequence contamination filtering, Sequencing quality control, Visualisation, Aggregation, Statistical calculation, Taxonomic classification</td>
      <td>2022-09-29</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>krakentools_kreport2krona, fastqc, fastp, porechop, kraken2, taxonomy_krona_chart, Filter1</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1249</td>
      <td>1962</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Variant calling, Genome assembly, Sequence assembly visualisation, Base-calling, De-novo assembly, Antimicrobial resistance prediction, Mapping assembly, Taxonomic classification, Multilocus sequence typing, Multiple sequence alignment, Validation, Scatter plot plotting, Visualisation, Cross-assembly, Aggregation, Pairwise sequence alignment, Data handling, Sequence alignment analysis, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree analysis, Phylogenetic tree generation (from molecular sequences), Phylogenetic tree generation, Sequence composition calculation, Box-Whisker plot plotting, Sequence assembly, Sequencing quality control, Phylogenetic tree reconstruction, Sequence contamination filtering, Statistical calculation, Mapping</td>
      <td>2023-01-26</td>
      <td>2024-09-27</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>regex1, regexColumn1, CONVERTER_gz_to_uncompressed, mlst, __BUILD_LIST__, Remove beginning1, fasta2tab, fastp, abricate, collapse_dataset, __FILTER_EMPTY_DATASETS__, samtools_depth, tab2fasta, tp_sorted_uniq, snpSift_filter, samtools_fastx, tp_replace_in_column, clair3, porechop, __FILTER_FAILED_DATASETS__, Paste1, ggplot2_heatmap, bandage_image, bcftools_norm, Count1, compose_text_param, krakentools_extract_kraken_reads, krakentools_kreport2krona, medaka_consensus_pipeline, minimap2, newick_display, clustalw, collection_element_identifiers, barchart_gnuplot, tp_find_and_replace, fasta_merge_files_and_filter_unique_sequences, collection_column_join, snpSift_extractFields, Grep1, add_line_to_file, bamtools_split_mapped, multiqc, kraken2, Add_a_column1, tp_cut_tool, Cut1, tp_split_on_column, bcftools_consensus, tp_multijoin_tool, samtools_coverage, bedtools_getfastabed, param_value_from_file, fastqc, nanoplot, table_compute, split_file_to_collection, fasttree, Grouping1, taxonomy_krona_chart, flye, tp_head_tool</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2001</td>
      <td>2968</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Sequence comparison, Taxonomic classification, Validation, Conversion, Visualisation, Sequence trimming, Phylogenetic analysis, Phylogenetic tree visualisation, Phylogenetic inference, Nucleic acid sequence analysis, Sequence alignment analysis, Species frequency estimation, Phylogenetic tree analysis, Primer removal, Sequence composition calculation, Phylogenetic tree editing, Sequencing quality control, Sequence similarity search, Read pre-processing, Statistical calculation</td>
      <td>2019-11-21</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>humann, bg_sortmerna, combine_metaphlan2_humann2, metaphlan, cutadapt, humann_renorm_table, graphlan_annotate, fastq_paired_end_interlacer, humann_unpack_pathways, export2graphlan, humann_regroup_table, tp_find_and_replace, Grep1, multiqc, Cut1, humann_split_stratified_table, fastqc, humann_rename_table, graphlan, taxonomy_krona_chart</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2147260</td>
      <td>5933139</td>
      <td>4.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Visualisation, Phylogenetic tree analysis, Aggregation, Statistical calculation, Nucleic acid sequence analysis</td>
      <td>2023-05-03</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>krakentools_kreport2krona, metaphlan, interactive_tool_phinch, interactive_tool_pavian, est_abundance, kraken_biom, __UNZIP_COLLECTION__, taxonomy_krona_chart, kraken2</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4540</td>
      <td>6233</td>
      <td>1.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Sequence comparison, Taxonomic classification, Validation, Conversion, Visualisation, Sequence trimming, Phylogenetic analysis, Phylogenetic tree visualisation, Phylogenetic inference, Nucleic acid sequence analysis, Sequence alignment analysis, Species frequency estimation, Phylogenetic tree analysis, Primer removal, Sequence composition calculation, Phylogenetic tree editing, Sequencing quality control, Sequence similarity search, Read pre-processing, Statistical calculation</td>
      <td>2020-02-13</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>humann, bg_sortmerna, combine_metaphlan2_humann2, metaphlan, cutadapt, humann_renorm_table, graphlan_annotate, fastq_paired_end_interlacer, humann_unpack_pathways, export2graphlan, humann_regroup_table, tp_find_and_replace, Grep1, multiqc, Cut1, humann_split_stratified_table, fastqc, humann_rename_table, graphlan, taxonomy_krona_chart</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>280</td>
      <td>357</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial Resistance</td>
      <td>Genome assembly, Sequence assembly visualisation, De-novo assembly, Mapping assembly, Sequence analysis, Box-Whisker plot plotting, Scatter plot plotting, Aggregation, Pairwise sequence alignment</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bandage_image, minimap2, unicycler, nanoplot, PlasFlow, gfa_to_fa, miniasm, racon, staramr_search</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1301</td>
      <td>1976</td>
      <td>4.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Sequence contamination filtering, Validation, Sequencing quality control, Visualisation, Sequence composition calculation, Statistical calculation, Taxonomic classification</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_replace_in_line, datamash_reverse, multiqc, fastqc, Remove beginning1, fastp, porechop, kraken2, taxonomy_krona_chart</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2802</td>
      <td>3894</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Variant calling, DNA barcoding, Visualisation, Deposition, Analysis</td>
      <td>2024-06-05</td>
      <td>2024-09-26</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>cat1, dada2_makeSequenceTable, dada2_filterAndTrim, __SORTLIST__, tp_replace_in_column, __UNZIP_COLLECTION__, tp_replace_in_line, dada2_assignTaxonomyAddspecies, dada2_mergePairs, phyloseq_from_dada2, dada2_plotQualityProfile, collection_element_identifiers, interactive_tool_phyloseq, dada2_learnErrors, dada2_removeBimeraDenovo, dada2_dada, Add_a_column1, dada2_seqCounts, Grouping1, tp_head_tool</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1162</td>
      <td>1566</td>
      <td>1.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Variant calling, Genome assembly, Sequence assembly visualisation, Sequence assembly validation, Visualisation, Sequence trimming, Local alignment, Sequence file editing, Data handling, Sequence alignment analysis, Primer removal, Formatting, Sequence composition calculation, Sequencing quality control, Read mapping, Read pre-processing, Sequence contamination filtering, Statistical calculation</td>
      <td>2022-12-05</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bandage_image, seqtk_subseq, megahit_contig2fastg, cutadapt, random_lines1, tp_cat, megahit, fastqc, bg_uniq, collection_column_join, bandage_info, metaspades, coverm_contig, bowtie2, filter_tabular, quast, bamtools, ngsutils_bam_filter</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1671</td>
      <td>2337</td>
      <td>2.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>136</td>
      <td>201</td>
      <td>3.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence assembly validation, Genome assembly, Validation, Sequencing quality control, Statistical calculation, Sequence composition calculation</td>
      <td>2023-12-05</td>
      <td>2024-09-12</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>megahit, checkm_lineage_wf</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2123</td>
      <td>2591</td>
      <td>1.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>154</td>
      <td>283</td>
      <td>3.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Sequence read processing, DNA barcoding, Phylogenetic tree analysis, Phylogenetic tree generation, Taxonomic classification, Sequencing quality control, Visualisation, Phylogenetic tree reconstruction, Phylogenetic analysis, Sequence clustering</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_make_contigs, mothur_chimera_vsearch, mothur_count_seqs, mothur_screen_seqs, mothur_cluster, mothur_classify_otu, mothur_summary_seqs, mothur_unique_seqs, mothur_dist_shared, mothur_make_biom, mothur_classify_seqs, mothur_dist_seqs, mothur_pre_cluster, XY_Plot_1, mothur_remove_seqs, mothur_remove_groups, mothur_sub_sample, newick_display, mothur_venn, mothur_rarefaction_single, mothur_tree_shared, mothur_summary_single, mothur_remove_lineage, mothur_filter_seqs, mothur_seq_error, mothur_get_groups, mothur_heatmap_sim, mothur_align_seqs, mothur_count_groups, mothur_taxonomy_to_krona, mothur_make_shared, mothur_cluster_split, taxonomy_krona_chart</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>5547</td>
      <td>8409</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Sequence read processing, DNA barcoding, Phylogenetic tree analysis, Phylogenetic tree generation, Taxonomic classification, Sequencing quality control, Visualisation, Phylogenetic tree reconstruction, Phylogenetic analysis, Sequence clustering</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_chimera_vsearch, mothur_count_seqs, mothur_screen_seqs, mothur_cluster, mothur_classify_otu, mothur_summary_seqs, mothur_unique_seqs, mothur_dist_shared, collapse_dataset, mothur_classify_seqs, mothur_dist_seqs, mothur_pre_cluster, XY_Plot_1, mothur_remove_seqs, mothur_remove_groups, mothur_sub_sample, newick_display, mothur_venn, mothur_rarefaction_single, mothur_tree_shared, mothur_summary_single, mothur_remove_lineage, mothur_filter_seqs, mothur_heatmap_sim, mothur_count_groups, mothur_make_shared, mothur_cluster_split</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2011</td>
      <td>3607</td>
      <td>5.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>interactive_tool_rstudio, upload1, raxml</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1379</td>
      <td>2079</td>
      <td>2.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Variant calling, Antimicrobial resistance prediction, Phylogenetic tree visualisation, Phylogenetic tree generation</td>
      <td>2022-03-16</td>
      <td>2024-07-12</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>samtools_view, tp_replace_in_line, tp_sed_tool, bcftools_consensus, snippy, snp_sites, collapse_dataset, trimmomatic, tp_cat, tp_easyjoin_tool, tp_grep_tool, Grep1, tb_variant_filter, addName, tb_profiler_profile, upload1, __MERGE_COLLECTION__, snp_dists</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1610</td>
      <td>2504</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial Resistance</td>
      <td>Genome visualisation, Read mapping, Genome annotation</td>
      <td>2024-01-23</td>
      <td>2024-09-27</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, Grep1, tbl2gff3, bakta, upload1, bowtie2, staramr_search</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1183</td>
      <td>1635</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Variant calling, Sequence alignment analysis, Antimicrobial resistance prediction, Sequence analysis, Phylogenetic tree generation, Genome visualisation, Sequence composition calculation, Taxonomic classification, Statistical calculation, Validation, Sequencing quality control, Global alignment, Sequence alignment, Sequence contamination filtering, Phylogenetic tree visualisation, Local alignment</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mosdepth, tp_sed_tool, qualimap_bamqc, jbrowse, snippy, jvarkit_wgscoverageplotter, samtools_stats, fastp, upload1, EMBOSS:%20seqret84, tp_awk_tool, tb_variant_filter, tb_profiler_profile, multiqc, kraken2, tbvcfreport, EMBOSS: seqret84, bcftools_consensus, __FLATTEN__, fastqc</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5800</td>
      <td>9186</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Sequence contamination filtering, Validation, Sequencing quality control, Visualisation, Sequence composition calculation, Statistical calculation, Taxonomic classification</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_replace_in_line, datamash_reverse, multiqc, fastqc, Remove beginning1, fastp, porechop, kraken2, taxonomy_krona_chart</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2802</td>
      <td>3894</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>sqlite_to_tabular, query_tabular, search_gui, peptide_shaker, unipept</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2320</td>
      <td>3839</td>
      <td>2.633333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Differential protein expression analysis, Filtering, Heat map generation, Indexing, Query and retrieval, Visualisation, Principal component visualisation, Quantification, Functional clustering, Statistical inference</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_db, metaquantome_stat, metaquantome_sample, metaquantome_viz, metaquantome_expand, metaquantome_filter</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>378</td>
      <td>510</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Differential protein expression analysis, Filtering, Heat map generation, Indexing, Query and retrieval, Visualisation, Principal component visualisation, Quantification, Functional clustering, Statistical inference</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_db, metaquantome_stat, metaquantome_sample, metaquantome_viz, metaquantome_expand, metaquantome_filter</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>353</td>
      <td>509</td>
      <td>3.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation, Filtering, Formatting, Label-free quantification</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, tp_replace_in_line, flashlfq, regex1, query_tabular, search_gui, peptide_shaker, Grep1, Remove beginning1, msconvert, tp_replace_in_column, unipept, Filter1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>782</td>
      <td>1355</td>
      <td>3.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Expression analysis, Sequence contamination filtering, Sequencing quality control, Cross-assembly, Sequence composition calculation, Statistical calculation, Taxonomic classification</td>
      <td>2024-07-15</td>
      <td>2024-09-02</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastqc, upload1, recentrifuge, est_abundance, fastp, kraken2</td>
      <td>UseGalaxy.eu</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>229</td>
      <td>296</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Variant calling, Data handling, Sequence alignment analysis, Genome indexing, Formatting, Sequence composition calculation, Generation, Sequencing quality control, Read mapping, Sequence alignment, Statistical calculation</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Cut1, bamFilter, fastqc, freebayes, picard_MergeSamFiles, vcf2tsv, bwa_mem, vcffilter2, picard_MarkDuplicates, bamleftalign</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7181</td>
      <td>13614</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Variant calling, Sequence alignment analysis, Antimicrobial resistance prediction, Sequence analysis, Phylogenetic tree generation, Genome visualisation, Sequence composition calculation, Taxonomic classification, Statistical calculation, Validation, Sequencing quality control, Global alignment, Sequence alignment, Sequence contamination filtering, Phylogenetic tree visualisation, Local alignment</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mosdepth, tp_sed_tool, qualimap_bamqc, jbrowse, snippy, jvarkit_wgscoverageplotter, samtools_stats, fastp, upload1, EMBOSS:%20seqret84, tp_awk_tool, tb_variant_filter, tb_profiler_profile, multiqc, kraken2, tbvcfreport, EMBOSS: seqret84, bcftools_consensus, __FLATTEN__, fastqc</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>5800</td>
      <td>9186</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Variant calling, Phylogenetic tree visualisation, Genome visualisation, Phylogenetic tree generation</td>
      <td>2018-02-26</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, snippy</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>12499</td>
      <td>20722</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>